submissions/aaronMubanga/database/init.sql

```diff
@@ -0,0 +1,104 @@
+
+-- creation of a DataBase
+
+CREATE DATABASE data_bases;
+
+-- Checking for success
+
+SHOW DATABASE;
+
+-- Selection of a DataBase to work on.
+
+USE data_bases;
+
+-- (1) Table Food Bag 
+
+CREATE TABLE food_bag (
+	id INT NOT NULL AUTO_INCREMENT,
+    type VARCHAR(100) NOT NULL,
+    PRIMARY KEY (id)
+);
+
+-- Addition of values to Food Bag
+
+INSERT INTO food_bag (type)
+	VALUE	('Fast Food'),
+			('Vegetables & Fruits'),
+            ('Drinks & Crocktails'),
+            ('Resturants')
+;
+
+-- Checking table Results
+
+SELECT * FROM food_bag;
+
+-- (3) Table Admins
+
+CREATE TABLE admins (
+	id INT NOT NULL AUTO_INCREMENT,
+    name VARCHAR(250) NOT NULL,
+    type ENUM('0', '1') NOT NULL,
+    email_address VARCHAR(250) NOT NULL,
+    phone_number INT NOT NULL,
```

Comment:
if someone is going to use +234 or +91 , it will be an issue with INT so change this VARCHAR (50)